docs/source/use_cases/annotate_variants/interval_joins.rst

```diff
@@ -110,162 +103,60 @@ Enter this query in the "Query Editor":
       Func,
       Gene,
       PopFreqMax,
-      ExonicFunc,
+      ExonicFunc
     FROM
-      [silver-wall-555:TuteTable.hg19]
+      `silver-wall-555.TuteTable.hg19`
     WHERE
-      PopFreqMax <= 0.01 ) AS annots
-  JOIN EACH (
-    SELECT
-      reference_name,
-      start,
-      reference_bases,
-      alternate_bases,
-      num_variant_alleles,
-      total_num_alleles,
-      vars.bin,
-    FROM (
-      SELECT
-        reference_name,
-        start,
-        END,
-        reference_bases,
-        NTH(1, alternate_bases) WITHIN RECORD AS alternate_bases,
-        SUM(call.genotype = 1) WITHIN RECORD AS num_variant_alleles,
-        SUM(call.genotype >= 0) WITHIN RECORD AS total_num_alleles,
-        # This must match the bin size in the UDF.
-        INTEGER(FLOOR(start / 50000)) AS bin,
-      FROM
-        [genomics-public-data:1000_genomes_phase_3.variants_20150220_release]) AS vars
-    JOIN (
-      SELECT
-        gene,
-        Chr,
-        region_start,
-        region_end,
-        bin,
-      FROM (binIntervals([myIntervalTable]))) AS intervals
-      # The JOIN criteria is complicated since we are trying to see if a SNP
-      # overlaps an interval.
-    ON
-      vars.reference_name = intervals.Chr
-      AND vars.bin = intervals.bin
-    WHERE
-      intervals.region_start <= vars.start
-      AND intervals.region_end >= vars.END) AS s_vars
-  ON
-    s_vars.reference_name = annots.Chr
-    AND s_vars.start = annots.Start
-    AND s_vars.reference_bases = annots.Ref
-    AND s_vars.alternate_bases = annots.Alt
-  GROUP BY
+      PopFreqMax <= 0.01 )
+    --
+    -- And finally JOIN the variants in the regions of interest
+    -- with annotations for rare variants.
+  SELECT
     Chr,
-    Start,
+    annots.Start AS Start,
     Ref,
-    Alt,
+    annots.Alt,
     Func,
     Gene,
     PopFreqMax,
     ExonicFunc,
+    num_variant_alleles,
+    total_num_alleles
+  FROM
+    rare_variant_annotations AS annots
+  JOIN
+    gene_variants AS vars
+  ON
+    vars.reference_name = annots.Chr
+    AND vars.start = annots.Start
+    AND vars.reference_bases = annots.Ref
+    AND vars.alt = annots.Alt
   ORDER BY
     Chr,
     Start
 
-The Tute table is for SNP annotation.  If a different table were used to annotate both SNPs and INDELs, you may wish to apply a similar UDF to also "bin" the rows from the variants table, since INDELs may span multiple bins.
-
-.. sidebar:: Bin Size
-
-  Notice the bin size of 50,000 base pairs.  This value can be changed, but be sure to change it in both the UDF and the SQL.
-
 Results
 -------
 
-A specific run of the above interval JOIN took `Query complete (223.9s elapsed, 3.38 TB processed)` on:
+A specific run of the above interval JOIN took `Query complete (92.1s elapsed, 3.38 TB processed)` on:
```

Comment:
I think this would read better as (just reformatting):

A specific run of the above interval JOIN took 

`Query complete (92.1s elapsed, 3.38 TB processed)`

on: